Makefile

```diff
@@ -25,8 +25,10 @@ ${FISSILE_BINARY}: bin/dev/install_tools.sh
 ########## VAGRANT VM TARGETS ##########
 
 certs: uaa-certs
+	[ -n ${DOMAIN} ] && sed -i "s/^DOMAIN=.*/DOMAIN=${DOMAIN}/g" ${GIT_ROOT}/bin/settings/network.env
 	${GIT_ROOT}/bin/generate-dev-certs.sh cf bin/settings/certs.env
 	${GIT_ROOT}/bin/settings/kube/ca.sh
+	git checkout ${GIT_ROOT}/bin/settings/network.env
```

Comment:
I object to throwing away the manual changes I have made to network.env

If you want to modify a file and reset it afterwards, please copy to a temp file instead.